### ISSUE TITLE
Shortcuts for subtask manipulation in task view

### ISSUE THREAD

**Issue by mgsloan** [mgsloan/todoist-shortcuts#94]
I have a change adding a fair bit of shortcuts for the new task view, but haven't yet added stuff for manipulating the subtasks.  This issue tracks doing that.

**Comment by antois** [comment]
Hi, thanks for the great work! 

Do you have any insights on the timeline of this feature? I am really missing this for navigating tasks recursively using only keyboard. This would speed things up so much and I am sure it's not only me.

**Comment by mgsloan** [comment]
Hey, I don't have a timeline for this.  Personally I rarely use task view or even nested tasks.  It does seem like a few people want this, though, so I implement it.  For those who code, I would merge a PR implementing this.

**Comment by mgsloan** [comment]
Closing as wontfix, I personally don't use the task view.

**Comment by onguarde** [comment]
the main highlight of this extension, is the ability to hover over any subtask using the cursor, and press v/t to move the task or schedule it quickly. 

it's only natural to extend the same shortcuts to the task view for consistency. now, we have to switch to a different mode, once we enter task view which is a pity.

**Comment by mgsloan** [comment]
@onguarde Glad you find some parts of todoist-shortcuts useful!

I wrote this extension to scratch my own itch back when Todoist had no native shortcuts.  I did a little exploratory hacking to see if it would be straightforward to add support for subtask manipulation.  The result is [a version of todoist-shortcuts](https://github.com/mgsloan/todoist-shortcuts/tree/task-view-subtasks)
 that lacks all normal task view shortcuts but does support what you ask for, with a bunch of other misc stuff broken.  I estimate that doing proper support would take another 5 hours at least, and that doesn't include the cost of ongoing maintenance.

Personally I don't use the task view and I don't quite understand why it exists.  I guess as an integrated place for comments and other things?  So, time and energy I put towards this would just be to help the users that desire this feature (which seems like few judging by this issue's activity).

[Open Source Maintainers Owe You Nothing | Mike McQuaid](https://mikemcquaid.com/open-source-maintainers-owe-you-nothing/) is a good treatment of this general topic.  I could be convinced to do it if users pool together $300 (which is an amazing deal when you work out the $/hr including ongoing maintenance)
